Haskell/Core/Simplicity/BitMachine/StaticAnalysis/Cost.hs

```diff
@@ -40,7 +40,7 @@ instance Assert TermWeight where
 instance Witness TermWeight where
   witness _ = result
    where
-    result = TermWeight $ fromInteger (bitSizeR (reifyProxy result))
+    result = TermWeight $ overhead + milli (fromInteger . bitSizeR $ reifyProxy result)
```

Comment:
9dcdc08d5d8b073b4eef729c4d239dd630e566d8: This could be `TermWeight $ overhead + milli (bitSizeR (reifyProxy result))` right? That would make it the same as the for `iden` and for `comp`.

Reply:
Fixed.